Plugin~/WebRTCPlugin/Codec/NvCodec/NvEncoder.cpp

```diff
@@ -85,7 +89,10 @@ namespace webrtc
         std::memcpy(&nvEncConfig, &presetConfig.presetCfg, sizeof(NV_ENC_CONFIG));
         nvEncConfig.profileGUID = NV_ENC_H264_PROFILE_BASELINE_GUID;
         nvEncConfig.gopLength = nvEncInitializeParams.frameRateNum;
-        nvEncConfig.rcParams.averageBitRate = m_bitrateAdjuster->GetAdjustedBitrateBps();
+        nvEncConfig.rcParams.rateControlMode = NV_ENC_PARAMS_RC_CBR_LOWDELAY_HQ;
+        nvEncConfig.rcParams.averageBitRate = (static_cast<unsigned int>(5.0f *
+            nvEncInitializeParams.encodeWidth *
+            nvEncInitializeParams.encodeHeight) / (1280 * 720)) * 100000;
```

Comment:
this value `(1280 * 720)`, could you use this filed value?

```suggestion
            nvEncInitializeParams.encodeHeight) / (m_width* m_height)) * 100000;
```

Reply:
Oh, I will fix it